.bumpversion.cfg

```diff
@@ -3,7 +3,7 @@ current_version = 1.10.2a0.dev0
 commit = True
 tag = False
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)((?P<pre>[a-z]+)(?P<num>\d+))?(\.(?P<release>dev)(?P<dev>\d+))?
-serialize = 
```

Comment:
Are we still using bumpversion??

Reply:
Haha...uuummm...no idea. I guess I'll say no and remove it.